library/std/src/net/mod.rs

```diff
@@ -33,6 +33,8 @@ pub use self::tcp::IntoIncoming;
 pub use self::tcp::{Incoming, TcpListener, TcpStream};
 #[stable(feature = "rust1", since = "1.0.0")]
 pub use self::udp::UdpSocket;
+#[unstable(feature = "anonymous_pipe", issue = "127154")]
+pub use crate::sys::anonymous_pipe::{pipe, PipeReader, PipeWriter};
```

Comment:
In the ACP this was in a new `pipe` module under `std::io`. `std::net` seems fine to me but this should maybe have its own `pipe` module anyway ?

Reply:
Yeah I can move it to a separate module.

Reply:
Moved it to new module `std::pipe`